data/ethereum/init.go

```diff
@@ -9,12 +9,14 @@ const (
 	BusyFinalizing
 	Finalized
 	Released
+	Failed
 
 	BROADCASTING  string = "broadcasting"
 	FINALIZING    string = "finalizing"
 	FINALIZE      string = "finalize"
 	MINTING       string = "minting"
 	CLEANUP       string = "cleanup"
+	CLEANUPFAILED string = "cleanupfailed"
```

Comment:
This cleanup function , only deletes jobs if they exist . If they dont it does not cause a run-time failure from storage